File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/Python/PRAK503-2310817310002-JovanGilbertNatamasindah.py
def maksimal(a,b) :

    if (a > b) :

        return a

    else :

        return b

def minimal(a,b) :

    if (a < b) :

        return a

    else :

        return b

batas = 0
maks = -100000
minim = 100000

bilangan = int(input("Masukkan bilangan : "))
nilai = list(map(int, input().split()))

while (batas < bilangan) :
    
    maks = maksimal(maks,nilai[batas])
    minim = minimal(minim,nilai[batas])
    batas += 1

    
print(f"bilangan maksimal dan minimum adalah : {maks} {minim}")

File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/C/PRAK503-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

int maksimal(int a, int b){

    if (a > b) {

        return a;
    }

    else  {

        return b;
    }

}

int minimal(int a, int b){

    if (a < b) {

        return a;
    }

    else {
        return b;
    }

}


int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;

    printf("Masukkan bilangan : ");
    scanf("%d", &bilangan);

    while(batas < bilangan){

        int nilai;

        scanf("%d", &nilai); 
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;

    }
    
    printf("bilangan maksimal dan minimum adalah : %d %d",maks,minim); 
}

